cmd/kubeseal/main_test.go

```diff
@@ -109,18 +109,23 @@ func TestParseKey(t *testing.T) {
 func TestOpenCertFile(t *testing.T) {
 	certFile := tmpfile(t, []byte(testCert))
 
-	f, err := openCert(certFile)
-	if err != nil {
-		t.Fatalf("Error reading test cert file: %v", err)
-	}
+	testCases := []string{"", "file://"}
+	for _, prefix := range testCases {
+		certURL := fmt.Sprintf("%s%s", prefix, certFile)
 
-	data, err := ioutil.ReadAll(f)
```

Comment:
Would it make sense to also test `http` URI? (dunno how much much code-vs-lib test coverage we'd be increasing)

Reply:
not sure; the only benefit I see would be to guard against us replacing the straight-forward `client.Get` call with something that treats `file://` specially and skips the `net/http` package. I don't find that possibility particularly likely, but you never know ... :-)

Reply:
anyway it turned out to be straightforward to test, so done